Hypnotic-Herons/sprint-3
# Db initial

## Link to Ticket
none

## Description of Proposed Changes
added folders for views, models, and forms and created __init__.py for each. 

## Steps to Test

fetch and open vs code to make sure folders are present. 

```
git fetch --all
git checkout db_inital
code .

```
